commerce-account-admin-web/src/main/resources/META-INF/resources/user_action.jsp

```diff
@@ -51,4 +56,47 @@ CommerceAccountUserRel commerceAccountUserRel = (CommerceAccountUserRel)row.getO
 			url="<%= deleteURL %>"
 		/>
 	</c:if>
-</liferay-ui:icon-menu>
\ No newline at end of file
+</liferay-ui:icon-menu>
+
+<aui:script use="liferay-item-selector-dialog">
+	$('#<portlet:namespace /><%= editUserRoleId %>').on(
+		'click',
+		function(event) {
+			event.preventDefault();
+
+			var itemSelectorDialog = new A.LiferayItemSelectorDialog(
+				{
+					eventName: 'userRoleItemSelector',
+					on: {
+						selectedItemChange: function(event) {
+						var <portlet:namespace />addUserRolesIds = [];
```

Comment:
tabbing: `var <portlet:namespace />addUserRolesIds` and `var selectedItems` should be indented one more tab.